async/unstable_mux_async_iterator.ts

```diff
@@ -40,134 +37,14 @@ interface TaggedYieldedValue<T> {
  *
  * @typeParam T The type of the provided async iterables and generated async iterable.
  */
-export class MuxAsyncIterator<T> implements AsyncIterable<T> {
-  #iteratorCount = 0;
-  #yields: Array<TaggedYieldedValue<T>> = [];
-  // deno-lint-ignore no-explicit-any
-  #throws: any[] = [];
-  #signal = Promise.withResolvers<void>();
-
+export class MuxAsyncIterator<T> extends MaxAsyncIterator_<T> {
```

Comment:
`MaxAsyncIterator_` this seems to be a typo of `MuxAsyncIterator_`?

Reply:
Ah, true. Thanks for pointing. Can you send a PR?

Reply:
opened #6096